docs/pages/en-us/Getting-Started.md

```diff
@@ -258,9 +258,10 @@ This object represents a logical connection to the datastore (that is, Elasticse
 
 The `AuditConnection` object, for direct to Elasticseach, can be constructed by setting the `CAF_AUDIT_MODE` environment variable to `elasticsearch` and then using one of the static `createConnection()` methods in the `AuditConnectionFactory` class:  
 - `createConnection()` method has no parameters and requires the following environment variables to be set when creating a connection for direct to Elasticsearch:
-        - `CAF_ELASTIC_HOST_AND_PORT_VALUES` refers to one or more of the nodes of the Elasticsearch cluster as a comma-separated list.
         - `CAF_ELASTIC_NUMBER_OF_SHARDS` the number of primary shards that an index should have. Defaults to 5.
         - `CAF_ELASTIC_NUMBER_OF_REPLICAS` the number of replica shards (copies) that each primary shard should have. Defaults to 1.  
+        - 'CAF_ELASTIC_HOST_VALUES': A comma separated list of hostnames to use when contacting elasticsearch. eg. localhost, otherHost
+        - 'CAF_ELASTIC_PORT_VALUE': The REST port of the ElasticSearch server listens on. e.g. 9200
```

Comment:
These config options don't look like the others.  Indeed the run into each other on the one line:
https://github.com/CAFAudit/audit-service/blob/fc2201587057f7cd7d200ec59dc50ed47f1313a4/docs/pages/en-us/Getting-Started.md#audit-connection

Reply:
Still looks really bad but at least it is consistent with what was there before.